packages/react-aria-components/src/Tree.tsx

```diff
@@ -267,23 +267,27 @@ export {_Tree as Tree};
 
 // TODO: readd the rest of the render props when tree supports them
 export interface TreeItemRenderProps extends Omit<ItemRenderProps, 'allowsDragging' | 'isDragging' | 'isDropTarget'> {
-  // Whether the tree row is expanded.
-  isExpanded: boolean
+  // Whether the tree item is expanded.
+  isExpanded: boolean,
+  // TODO: api discussion, how do we feel about the below? This is so we can still style the row as grey when a child element within is focused
+  // Maybe should have this for the other collection item render props
+  // Whether the tree item's children have keyboard focus.
+  isFocusVisibleWithin: boolean
```

Comment:
Added so the treeview's row can retain its "focused" background color when focus moves to one of the elements within the row. I'll clean up the comment if we think this is ok

Reply:
makes sense to me
i'm fine with the name as well

I'm surprised we don't have this on Table RAC, meanwhile, TableView has this kind of styling which means we'll need this in Table as well

Reply:
Yeah, we'll probably want these for other collection components, I'll make a followup ticket for this